applications/nrf_desktop/src/hw_interface/battery_charger.c

```diff
@@ -113,16 +113,20 @@ static int cso_pin_control(bool enable)
 
 static int charging_enable(void)
 {
+	u32_t charging = CONFIG_DESKTOP_BATTERY_CHARGER_ENABLE_INVERSED ? 0 : 1;
+
 	return gpio_pin_write(gpio_dev,
 			      CONFIG_DESKTOP_BATTERY_CHARGER_ENABLE_PIN,
-			      1);
+			      charging);
 }
 
 static int charging_disable(void)
 {
+	u32_t charging = CONFIG_DESKTOP_BATTERY_CHARGER_ENABLE_INVERSED ? 1 : 0;
+
 	return gpio_pin_write(gpio_dev,
 			      CONFIG_DESKTOP_BATTERY_CHARGER_ENABLE_PIN,
-			      0);
+			      charging);
```

Comment:
Please have charging_enable and charging_disable call a common function that takes a bool as an argument to reduce the copy-paste here.

Reply:
ok